pyomo/gdp/plugins/hull.py

```diff
@@ -64,7 +64,7 @@ class ConvexHull_Transformation(Transformation):
         list of blocks and Disjunctions [default: the instance]
 
     The transformation will create a new Block with a unique
-    name beginning "_pyomo_gdp_chull_relaxation".  That Block will
+    name beginning "_pyomo_gdp_hull_relaxation".  That Block will
```

Comment:
While we are renaming things, this should be renamed `_pyomo_gdp_hull_reformulation`